### ISSUE TITLE
go to next/previous in accessible view for inline completions doesn't work

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#221521]
I noticed this when working on #221497, which has a different cause I am currently fixing. Wanted to note that it doesn't work _before_ my changes here 😄 

https://github.com/user-attachments/assets/2777805a-2dff-4751-a5be-3da51a7e2194

**Comment by meganrogge** [comment]
We weren't calling `const state = this._model.state.get();`, so it was using the old state.
